docs/content/topics/operators/storage.md

```diff
@@ -9,189 +9,169 @@ aliases:
   - /topics/storage.md
 ---
 
-TODO: update per v2 (no longer any job cache)
+# Storage in Brigade
 
-# How Brigade uses Kubernetes Persistent Storage
+Brigade utilizes storage for its in the following ways:
 
-Brigade allows script authors to declare two kinds of storage:
+  * [Shared Worker storage](#shared-worker-storage) wherein a Brigade Worker's
+    workspace may be shared with its Job(s). Optional; not enabled by default.
+  * [Artemis storage](#artemis-storage) for Brigade's Messaging/Queue component
+  * [MongoDB storage](#mongodb-storage) for Brigade's backing data store
 
-- per-job caches, which persist across builds
-- per-build shared storage, which exists as long as the build is running
+## Shared Worker storage
 
-Usage of these is described within the [JavaScript docs](javascript.md) and the
-[scripting guide](scripting.md).
+The workspace for a Brigade Worker can be shared with all Worker Jobs. This is
+an opt-in feature and isn't enabled by default. When enabled, Brigade will
+create a [PersistentVolume] on the underlying Kubernetes substrate and
+automatically add the corresponding volume mount to each Worker Job created.
```

Comment:
Jobs also have to opt-in by specifying a mount path. If they don't, they don't get access to the shared storage.

Reply:
Nevermind... you cover this pretty well further on down. I don't think we need to say it here also.